internal/config/subnet/api-key.go

```diff
@@ -59,13 +73,28 @@ func validateLicenseFormat(lic string) error {
 	return err
 }
 
+func validateAPIKeyFormat(apiKey string) error {
+	if len(apiKey) == 0 {
+		return nil
+	}
+
+	// Verify that the api key is a valid UUID string
+	_, err := uuid.Parse(apiKey)
+	return err
+}
+
 // LookupConfig - lookup config and override with valid environment settings if any.
 func LookupConfig(kvs config.KVS) (cfg Config, err error) {
 	if err = config.CheckValidKeys(config.SubnetSubSys, kvs, DefaultKVS); err != nil {
 		return cfg, err
 	}
 
 	cfg.License = env.Get(config.EnvMinIOSubnetLicense, kvs.Get(config.License))
+	err = validateLicenseFormat(cfg.License)
+	if err != nil {
+		return cfg, err
+	}
```

Comment:
if it is already set we do not need to validate this since we do not allow setting licenses anymore. WDYT?

Reply:
Removed